guides/source/engines.md

```diff
@@ -638,6 +638,7 @@ you will need to specify the `:path` option in your `Gemfile`:
 gem "blorgh", path: "engines/blorgh"
 ```
 
+Create an `engines` directory inside `unicorn` and copy the `blorgh` engine into it.  
```

Comment:
> you will need to specify the `:path` option in your `Gemfile`

It was probably not clear, but this sentence means that you need to point the path where you created the blorgh engine. We can probably clarify the sentence above but there is no need to copy/paste.